src/commands/pubsub-subscribers.json

```diff
@@ -0,0 +1,46 @@
+{
+  "SUBSCRIBERS": {
+    "summary": "Get the subscribers for each pubsub subscription type, filter by channel name.",
+    "complexity": "O(N) for the subcommand",
+    "group": "pubsub",
+    "since": "7.2.0",
+    "arity": -2,
+    "container": "PUBSUB",
+    "function": "pubsubCommand",
+    "command_flags": [
+      "PUBSUB",
+      "LOADING",
+      "STALE"
+    ],
+    "arguments": [
+      {
+        "name": "type",
+        "type": "oneof",
+        "arguments": [
+            {
+                "name": "global",
+                "type": "pure-token",
+                "token": "global"
```

Comment:
Patterns are also global. The naming of "channel", "pattern", "shard-channel" make the most sense to me.

Reply:
Oran was also suggestion in another PR `global` word is something which isn't coined anywhere. I find the naming above fine, will update.

Reply:
https://github.com/redis/redis/pull/8860#issuecomment-1501078129